adding twirf #7

gnunicorn opened the pull request:
- Speaker announcement
  - ViewSource Collaboration announcement
  - mini update on tickets (1st week of Aug)

I am tired and can't review it properly anymore. Input highly appreciated!

Rendered:
![screen shot 2016-07-22 at 18 34 27-fullpage](https://cloud.githubusercontent.com/assets/40496/17064131/40edfb1c-503b-11e6-830a-2c7d4f8755c4.png)

Commit:
add post

badboy commented on _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md:
```diff
@@ -0,0 +1,39 @@
+
+## First talks up
+
+As already mentioned last week, we've closed the CfP and by now informed everyone about their participation. At the moment we are still in process to gathering all details and ensuring the logistics for everyone, but the basic talk schedule is clear now. Which allows us to announce the first set of  confirmed speakers and their [talks](/talks/): 🎉
```
"still in process **of** gathering"

badboy commented on _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md:
```diff
@@ -0,0 +1,39 @@
+</div>
+
+
+(From right to left) **Stephan Hügel** will be sharing with us how he taught [himself Rust through the medium of coordinate system transforms and FFI](/talks/#teaching-myself-rust-through-the-medium-of-coordinate-system-transforms-and-ffi), while **Sonja Heinen** takes us a little off the beating past and asks us to ponder with her about [Hobby-oriented Programming](/talks/#hobby-oriented-program ...
```
one "programming" to many
Maybe

> takes us into the depth of programming the Internet of Things with ..

badboy commented on _posts/2016-07-22-this-week-in-rustfest-7-first-talks-and-viewsource.md:
```diff
@@ -0,0 +1,39 @@
+
+## Collaborating with View Source
+
+We are super please to announced we'll be collaborating with the awesome View Source Conference, which takes place earlier the same week as RustFest – 12-14th of September. Check it out here:
```
→ We are super pleased to announce ...

d moved to `please`, removed from `announced`

badboy commented:
Tiny little remarks, otherwise good to go

Commit:
It's a new day

https://www.youtube.com/watch\?v\=ILObfEzX92k

Commit:
from to to of.

https://www.youtube.com/watch\?v\=FTQbiNvZqaY

from *to to*... get it?

Commit:
better text for the IoT

![](http://annagerber.github.io/talks/images/connectallthethings.png)

Commit:
move the d

![](http://tt.imageshare.s3.amazonaws.com/critters/http-cat-codes/301-moved-permanently.jpg)

gnunicorn commented:
sorry. got distracted watching terrible trailers on youtube. merging now.

badboy commented:
Sounds like fun, care to share?^^

gnunicorn commented:
@badboy for sure:
- https://www.youtube.com/watch?v=_jM4lLYhjQs
- https://www.youtube.com/watch?v=jxKyDxthvOI
- https://www.youtube.com/watch?v=AfuVgfuaTbQ
- https://www.youtube.com/watch?v=W61-oe2g_qM
- https://www.youtube.com/watch?v=t0hSzMVVKC8

and for some reason:
- https://www.youtube.com/watch?v=s_BzTJ7etkA

gnunicorn commented:
I started the first because of [review I came across when looking for the 301 moved](https://thatdutchfilmlover.wordpress.com/2014/02/09/nurse-3d-2013/) –  one that made it sound super terrible. But it said "gore" and I was like ... I could do some gore...
